src/Resources/config/block.php

```diff
@@ -23,7 +24,9 @@
     $containerConfigurator->services()
 
         ->set('sonata.admin.block.admin_list', AdminListBlockService::class)
+            // NEXT_MAJOR: Remove public and sonata.container.private tag.
             ->public()
+            ->tag(AliasDeprecatedPublicServicesCompilerPass::PRIVATE_TAG_NAME, ['version' => '3.x'])
```

Comment:
Based on the comment you made, looks like block services need to be public, I've created https://github.com/sonata-project/SonataBlockBundle/issues/876 to track that.

In the meanwhile, we need block services to be public, so please remove the tags from `block.php`.